4-Deployment/README.md

```diff
@@ -0,0 +1,206 @@
+# Deploy your Angular application to Azure Cloud and use Azure services to manage your operations
+
+ 1. [Overview](#overview)
+ 1. [Scenario](#scenario)
+ 1. [Prerequisites](#prerequisites)
+ 1. [Registration](#registration)
+ 1. [Deployment](#deployment)
+ 1. [Explore the sample](#explore-the-sample)
+ 1. [More information](#more-information)
+ 1. [Community Help and Support](#community-help-and-support)
+ 1. [Contributing](#contributing)
+ 1. [Code of Conduct](#code-of-conduct)
+
+## Overview
+
+This sample demonstrates how to deploy an Angular single-page application (SPA) coupled with a .NET Core web API to **Azure Cloud** using [Azure Storage](https://docs.microsoft.com/azure/storage/blobs/) and [Azure App Service](https://docs.microsoft.com/azure/app-service/), respectively. To do so, we will use the [same code sample from Chapter 3](../3-Authorization-II/1-call-api).
+
+> :information_source: You can choose to deploy the [B2C sample from Chapter 3](../3-Authorization-II/1-call-api-b2c) instead. Steps below are more or less the same. 
+
+## Scenario
+
+1. The client application uses **MSAL Angular** to sign-in a user and obtain a JWT **Access Token** from **Azure AD**.
+1. The **Access Token** is used as a **bearer** token to *authorize* the user to call the protected web API.
+1. The protected web API responds with the claims in the **Access Token**.
+
+![Overview](./ReadmeFiles/topology.png)
+
+## Prerequisites
+
+- [VS Code Azure Tools Extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-node-azure-pack) extension is recommended for interacting with **Azure** through VS Code interface.
+- An **Azure subscription**. This sample uses **Azure Storage** and **Azure App Service**.
+
+## Setup
+
+### Step 1: Clone or download this repository
+
+From your shell or command line:
+
+```console
+    git clone https://github.com/Azure-Samples/ms-identity-javascript-angular-tutorial.git
+```
+
+or download and extract the repository .zip file.
+
+> :warning: To avoid path length limitations on Windows, we recommend cloning into a directory near the root of your drive.
+
+### Step 2: Install project dependencies
+
+- Setup the service app:
+
+```console
+    cd ms-identity-javascript-angular-tutorial
+    cd 3-Authorization-II/1-call-api
+    cd API
+    npm install
+```
+
+- Setup the client app:
+
+```console
+    cd ..
+    cd SPA
+    npm install
+```
+
+## Registration
+
+### Register the service app (.NET Core web API)
+
+Use the same app registration credentials that you've obtained during [**chapter 3-1**](../3-Authorization-II/1-call-api/README.md#registration). Update your project files here as needed.
+
+### Register the client app (Angular SPA)
+
+Use the same app registration credentials that you've obtained during [**chapter 3-1**](../3-Authorization-II/1-call-api/README.md#registration). Update your project files here as needed.
+
```

Comment:
We can add specific scetions:
For service: ../3-Authorization-II/1-call-api/README.md#register-the-service-app-msal-dotnet-api
For client: ../3-Authorization-II/1-call-api/README.md#register-the-client-app-msal-angular-spa